tests/unittest.cpp

```diff
@@ -1727,38 +1727,31 @@ TEST_CASE("stream_response")
 
       //consuming the headers, since we don't need those for the test
       static char buf[2048];
-      c.receive(asio::buffer(buf, 2048));
+      size_t headers_received = 0;
+      // magic number is 102 (it's the size of the headers, which is how much this line below needs to read)
+      while (headers_received < 102)
```

Comment:
It would name that number.
But shouldn't the `1250000` inside the while loop also be changed then, to be composed of the number of repetitions and the size of the string?

Reply:
I guess `1250000` shouldn't be changed since it's only used once, while 102 is used twice.

Reply:
It is used for generating the key_response and for checking the response. Two uses.

Reply:
well, `250000` is being used for generating `key_response` which is different..